apps/server/src/api-integration/integration.utils.ts

```diff
@@ -16,9 +17,35 @@ const whitelistedPayload = {
   skip: coerceBoolean,
 
   colour: coerceColour,
+
+  custom: coerceCustom,
 };
 
+function coerceCustom(value: unknown): EventCustomFields {
+  const customValue = value as { value: unknown; customProperty: string; type: 'string' };
+  switch (customValue.type) {
+    case 'string':
+      return {
+        [customValue.customProperty]: {
+          value: coerceString(customValue.value,
+          ),
+        },
+      };
+  }
+  throw new Error('Unable to parese custom value');
+}
+
 export function parse(property: string, value: unknown) {
+  if (property.startsWith('custom:')) {
+    const customProperty = property.split(':')[1];
```

Comment:
I believe we have existing logic for checking this? (Or similar)

Reply:
can't  find this
do you know where i should look?

Reply:
hmmm, i was so sure I wrote this logic but couldnt find it, sorry about this